views/index.njk

```diff
@@ -5,36 +5,78 @@
 {% endblock %}
 
 {% block content %}
-      <section class="hero d-flex justify-content-center mt-5">
+{% if theme == 'christmas' %}
```

Comment:
Componentize this into different views, one for christmas, halloween and default

Reply:
done